RecipeApp/Modules/RecipeBook/ViewModel/RecipeBookViewModel.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  RecipeBookViewModel.swift
+//  RecipeApp
+//
+//  Created by Hadley Igoe on 2019-12-17.
+//  Copyright © 2019 Hadley Igoe. All rights reserved.
+//
+
+import Foundation
+
+class RecipeBookViewModel {
+    
+    var favoriteService = FavoriteService.sharedService
+    var favoriteRecipes: [Meal]?
+    var updateHandler: (() -> Void)?
+    var completion: ((Meal) -> Void)?
```

Comment:
I would prefer to see this called something that describes its actual function. Completion would be appropriate if it was being called when closing or done. Something like "mealSelected" or something to that effect?